src/main/kotlin/com/example/noswimmingserver/global/security/SecurityConfig.kt

```diff
@@ -30,6 +31,9 @@ class SecurityConfig(
         http
             .authorizeRequests()
 
+            .antMatchers(HttpMethod.GET,"/auth/link").permitAll()
+            .antMatchers(HttpMethod.GET,"/auth/google").permitAll()
+
```

Comment:
🚫 **[ktlint]** <no-blank-lines-in-chained-method-calls> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Needless blank line(s)